gitea/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "arch": [
+    "aarch64",
+    "amd64"
+  ],
+  "codenotary": "alexandrep.github@gmail.com",
+  "description": "Gitea for Home Assistant",
+  "image": "ghcr.io/alexbelgium/gitea-{arch}",
+  "init": false,
+  "map": [
+    "share:rw",
+    "ssl:ro"
+  ],
+  "name": "Gitea",
+  "options": {
+    "certfile": "fullchain.pem",
+    "keyfile": "privkey.pem",
+    "ssl": false
+  },
+  "ports": {
+    "3000/tcp": 3000,
+    "2222/tcp": 2222
+  },
+  "ports_description": {
+    "3000/tcp": "Http web interface",
+    "22/tcp": "Ssh"
+  },
+  "schema": {
+    "certfile": "str",
+    "keyfile": "str",
+    "ssl": "bool"
+  },
+  "slug": "gitea",
+  "url": "https://github.com/baldarn/hassio-addons/tree/master/gitea",
```

Comment:
hi, your url is still mapped to your repo and not mine (if you prefer the addon to be added in my repo)

Reply:
yes, I did that to test it on my home assistance instance